reference-implementation/test/readable-byte-stream.js

```diff
@@ -1,25 +1,13 @@
 const test = require('tape-catch');
 
 test('ReadableByteStream can be constructed with no errors', t => {
-  new ReadableByteStream();
-
-  t.doesNotThrow(() => new ReadableByteStream(), 'ReadableByteStream constructed with no parameters');
-  t.doesNotThrow(() => new ReadableByteStream({ }),
-                 'ReadableByteStream constructed with an empty object as parameter');
-  t.doesNotThrow(() => new ReadableByteStream(undefined),
-                 'ReadableByteStream constructed with undefined as parameter');
-  let x;
-  t.doesNotThrow(() => new ReadableByteStream(x),
-                 'ReadableByteStream constructed with an undefined variable as parameter');
+  t.doesNotThrow(() => new ReadableStream({ pullInto() {} }),
+                 'ReadableByteStream constructed with an empty underlying byte source object as parameter');
   t.end();
 });
 
 test('ReadableByteStream: getReader(), then releaseLock()', t => {
-  const rbs = new ReadableByteStream({
-    pull() {
```

Comment:
Yes. Before this merge, the RBS had a behavior different from RS. The RBS invoked pull only in response to read() call. Now, it's given the queuing strategy mechanism. The HWM is set to 1, and so it invokes pull automatically once start completes and when the queue size is less than 1.